plugin-server/src/worker/tasks.ts

```diff
@@ -29,10 +29,6 @@ export const workerTasks: Record<string, TaskRunner> = {
     pluginScheduleReady: (hub) => {
         return hub.pluginSchedule !== null
     },
-    runLightweightCaptureEndpointEventPipeline: async (hub, args: { event: PipelineEvent }) => {
-        const runner = new EventPipelineRunner(hub, args.event)
-        return await runner.runLightweightCaptureEndpointEventPipeline(args.event)
-    },
     runEventPipeline: async (hub, args: { event: PluginEvent }) => {
```

Comment:
this needs to be `PipelineEvent` based on my reading - I don't like our names for the types either 😅